### DISCUSSION TITLE
(OneDrive Client for Linux) is disabled

### DISCUSSION THREAD

**Discussion by Peggasus6049** [abraunegg/onedrive#2025] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/108441560/176574564-b01d8fa8-e857-4a88-8848-4bf8296cc1c5.png)
I had to sign in to my account again which I've had to do before but last time it was fine. This time, after entering the authorisation url, I tried to login and received this error: AADSTS7000112: Application 'd50ca740-c83f-4d1b-b616-12c519384f0c'(OneDrive Client for Linux) is disabled.
It is a business account but I have been able to use this for a few months now so I don't know what is different this time.

**Comment by abraunegg** [comment, marked as answer]
@Peggasus6049 
> It is a business account but I have been able to use this for a few months now so I don't know what is different this time.

Your Active Directory Administrator has disabled the prior authorisation of the application.

You need to speak to who ever looks after your Office 365 environment.

This is nothing that I can solve.

**Reply by Peggasus6049** [reply]
I was worried that was the case. That's very annoying, I will ask them about it and see if they can enable it again. I'm really not sure what would have changed.
Thanks for your help!

**Comment by sumantagogoi** [comment]
when do we gett 22.04 jammy repo?

**Reply by abraunegg** [reply]
@sumantagogoi 
Your question has zero relevance to this thread .. please start a new discussion.
